.github/workflows/chatops.yml

```diff
@@ -33,12 +33,20 @@ jobs:
             })
             console.log(pr.data.head.ref)
             return pr.data.head.ref
-            
+      
+      - name: Generate GitHub token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
```

Comment:
Maybe we should go with [`v2`](https://github.com/tibdex/github-app-token/releases/tag/v2.0.0), especially if it revokes the created token at the end of the job with a [post script](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runspost).